recipes/serializers.py

```diff
@@ -9,3 +9,15 @@ class IngredientSerializer(serializers.ModelSerializer):
     class Meta:
         model = Ingredient
         fields = ('id', 'name', 'image')
+
+
+class RecipeSerializer(serializers.ModelSerializer):
+    image = Base64ImageField(required=False)
+    author = serializers.ReadOnlyField(source='author.username')
```

Comment:
You can use `PresentablePrimaryKeyRelatedField` of `django-extra-field` to serialize `author`.